src/com/boxboat/jenkins/library/notify/SlackWebHookNotifyTarget.groovy

```diff
@@ -19,15 +20,15 @@ class SlackWebHookNotifyTarget extends BaseConfig<SlackWebHookNotifyTarget> impl
                 color = "#36a64f"
                 break
         }
-        Config.pipeline.withCredentials([
-                Config.pipeline.string(credentialsId: credential, variable: 'SLACK_URL',)
-        ]) {
+
+
+        Closure closure = {
             String jsonStr = new JsonBuilder([
-                    text: "*${Config.pipeline.env.JOB_NAME}* (<${Config.pipeline.env.BUILD_URL}|build #${Config.pipeline.env.BUILD_NUMBER}>)",
+                    text       : "*${Config.pipeline.env.JOB_NAME}* (<${Config.pipeline.env.BUILD_URL}|build #${Config.pipeline.env.BUILD_NUMBER}>)",
```

Comment:
Space cleanup

Reply:
Result of IntelliJ code formatting. Are there different settings I should use for this formatting?
> ... 
> Could you run code format on modified files - in IntelliJ it is Ctrl+Alt+L on Linux (not sure about mac)

Reply:
intellij formatting :dollar: